charts/astronomer/values.yaml

```diff
@@ -18,12 +18,12 @@ images:
     tag: 0.16.3
     pullPolicy: IfNotPresent
   registry:
-    repository: quay.io/astronomer/ap-registry
-    tag: 3.11.5-4
+    repository: astronomerinc/ap-registry
+    tag: 3.14.2-2
     pullPolicy: IfNotPresent
   houston:
-    repository: quay.io/astronomer/ap-houston-api
-    tag: 0.16.18
+    repository: asoldatenko/ap-houston-api
```

Comment:
and here?

Reply:
temp fix to run it in CI, until houston PR in review https://github.com/astronomer/houston-api/pull/813